assemblies/assembly-getting-started-kogito-microservices.adoc

```diff
@@ -33,6 +33,21 @@ include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservice-runnin
 
 include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservice-interacting-app.adoc[leveloffset=+1]
 
+include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservices-deploying-on-openshift.adoc[leveloffset=+1]
+include::{enterprise-dir}/getting-started-kogito/con-kogito-microservices-on-ocp.adoc[leveloffset=+2]
+include::{enterprise-dir}/getting-started-kogito/con-kogito-operator-deployment-options.adoc[leveloffset=+2]
+include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-git-build.adoc[leveloffset=+3]
+include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-binary-build.adoc[leveloffset=+3]
+include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-custom-image-build.adoc[leveloffset=+3]
+include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-file-build.adoc[leveloffset=+3]
+//include::{enterprise-dir}/getting-started-kogito/con-file-build-operator-deployment-option.adoc[leveloffset=+3]
+//include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console.adoc[leveloffset=+2]
+include::{enterprise-dir}/getting-started-kogito/con-kogito-probes.adoc[leveloffset=+2]
+include::{enterprise-dir}/getting-started-kogito/proc-kogito-enable-probes-quarkus.adoc[leveloffset=+3]
+include::{enterprise-dir}/getting-started-kogito/proc-kogito-enable-probes-springboot.adoc[leveloffset=+3]
```

Comment:
on RHPAM Kogito Operator, Spring Boot Actuator is not handled, see https://github.com/kiegroup/kogito-operator/blob/1.5.x/core/kogitoservice/probe.go#L34
So I would not put that one as part of the doc in 7.11.0
@sutaakar can you confirm ?

Reply:
Well, it is not needed for 7.11 but I think we can leave it here, so customers don't have to add it when migrating to 7.12.